spec/fixtures/katello-certs-check/create_cert.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Generate CA
+openssl genrsa -out ca.key 2048
+openssl req -x509 -new -nodes -key ca.key -sha256 -days 3650 -out cacert.crt
+
+# Generate server certificate
+openssl genrsa -out foreman.example.com.key 2048
+openssl req -new -key foreman.example.com.key -out foreman.example.com.csr
```

Comment:
Should the .key and .csr files be in `.gitignore`? Are there other files that should be ignored, like a serial file?

Reply:
Done